settings/provisioner/foreman/default.ini

```diff
@@ -0,0 +1,3 @@
+[foreman]
```

Comment:
This is no longer required. we generate defaults based on `default` arguments in spec file